documentation/modules/cruise-control/proc-approving-optimization-proposal.adoc

```diff
@@ -34,48 +34,53 @@ To do so, refresh the optimization proposal to make sure it uses the latest clus
 +
 [source,shell,subs="+quotes"]
 ----
-kubectl annotate kafkarebalance _rebalance-cr-name_ strimzi.io/rebalance=refresh
+kubectl annotate kafkarebalance _<kafka_rebalance_resource_name>_ strimzi.io/rebalance=refresh
 ----
 
-.. Check the status of the `KafkaRebalance` resource:
+. Check the status of the optimization proposal:
 +
 [source,shell,subs="+quotes"]
 ----
-kubectl describe kafkarebalance _rebalance-cr-name_
+kubectl get kafkarebalance -o wide -w -n _<namespace>_
 ----
-
-.. Wait until the status changes to `ProposalReady`.
++
+--
+`PendingProposal`:: A `PendingProposal` status means the rebalance operator is polling the Cruise Control API to check if the optimization proposal is ready.
+`ProposalReady`:: A `ProposalReady` status means the optimization proposal is ready for review and approval.
+--
++
+When the status changes to `ProposalReady`, the optimization proposal is ready to approve.
 
 . Approve the optimization proposal that you want Cruise Control to apply.
 +
-Annotate the `KafkaRebalance` resource in Kubernetes:
+Annotate the `KafkaRebalance` resource in Kubernetes with `approve`:
```

Comment:
saying just "with `approve`" doesn't sound correct to me because an annotation is a key=value pair not just the value.

Reply:
Changed to:

> Annotate the `KafkaRebalance` resource in Kubernetes with `strimzi.io/rebalance=approve`: